### PR TITLE
Dev 4

### PR DESCRIPTION
**Pull request by ohatsenko**
Added custom validation and CustomGlobalExceptionHandler

### CODE REVIEW

File: src/main/resources/db/changelog/changes/001-create-books-table.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 databaseChangeLog:
   - changeSet:
       id: create-books-table
-      author: ivan
+      author: ohatsenko
```
Comment (JeniaSan) [inline src/main/resources/db/changelog/changes/001-create-books-table.yaml:4]: copy paste is evil)

**Review by JeniaSan — APPROVED** [review]
GJ!

File: src/main/resources/application.properties [hunk 2]
```diff
@@ -2,7 +2,7 @@ spring.datasource.url=jdbc:mysql://localhost:3306/bookstore?serverTimezone=UTC
 spring.datasource.username=root
 spring.datasource.password=Alex1234
 spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
-spring.jpa.hibernate.ddl-auto=validate
+spring.jpa.hibernate.ddl-auto=create-drop
```
Comment (Vasul-Pankiv) [inline src/main/resources/application.properties:5]: why you change it on create-drop?

**Review by Vasul-Pankiv — APPROVED** [review]

File: src/main/resources/db/changelog/changes/001-create-books-table.yaml (hunk 1 above)
Comment (ohatsenko) [reply src/main/resources/db/changelog/changes/001-create-books-table.yaml:4]: shit happens)

**Pull request merged**